extensions/GlobalMessages/GlobalMessagesT.i18n.php

```diff
@@ -261,6 +262,7 @@
 	'tog-previewonfirst' => 'Toggle option used in [[Special:Preferences]].',
 	'tog-nocache' => "[[Special:Preferences]], tab 'Misc.'. Offers the user the option of disabling caching of pages in the browser",
 	'tog-enotifwatchlistpages' => 'Option in [[Special:Preferences]] > {{int:prefs-personal}} > {{int:email}}.',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Option in [[Special:Preferences]] > {{int:prefs-personal}} > {{int:email}}." 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)